python/graphstorm/model/node_glem.py

```diff
@@ -156,10 +156,13 @@ def inference_route_is_gnn(self):
         # GLEM is being trained: use gnn route if not training lm
         return not self.training_lm
 
-    def init_optimizer(self, lr, sparse_optimizer_lr, weight_decay, lm_lr=None):
+    def init_optimizer(self, lr, sparse_optimizer_lr=None, weight_decay=0, lm_lr=None):
         """Initialize optimzer, which will be stored in self.lm._optimizer, self.gnn._optimizer
         """
         sparse_params = self.lm.get_sparse_params()
+        # check and set the sparse optimizer learning rate
+        if sparse_optimizer_lr is None:
```

Comment:
The sparse_optimizer_lr should have been set to lr here: https://github.com/awslabs/graphstorm/blob/2bf5c404348a1ad7805150426fa5748f60e923aa/python/graphstorm/config/argument.py#L1169
Also same as the previous

Reply:
updated.